dev/automate_training.py

```diff
@@ -112,61 +121,63 @@ def test_worker(config):
 
     # Step 1 : batch size, initial LR and LR scheduler
 
+    ### Training parameters
+    category = "training_parameters"
+    base_item = initial_config[category]
+    keys = ["batch_sizes", "initial_lrs", "lr_scheduler", "loss", "mixup_alpha"]
+
     batch_sizes = [8, 16, 32, 64]
     initial_lrs = [1e-2, 1e-3, 1e-4, 1e-5]
-
-    """
     lr_schedulers = [{"name": "CosineAnnealingLR"},
                     {"name": "CosineAnnealingWarmRestarts", "T_0": 10}
```

Comment:
@olix86 now that this is uncommented I have errors (X, Y not defined + missing comma). Should we recomment these lines?

Reply:
It's possible, there are other errors also, I'm still working on it (the way it calls cmd_train, etc.)

Reply:
I have slightly modified the thing about BCE: [here](https://github.com/neuropoly/ivado-medical-imaging/pull/239/commits/ec29f82da7637b61d9d88bd6f1afd8c6f35cf037)